### PR TITLE
Fix image_map example template

### PR DESCRIPTION
**Pull request by luca-rath**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #issuenum
| Related issues/PRs | #issuenum
| License | MIT
| Documentation PR | sulu/sulu-docs#prnum

#### What's in this PR?

Fixes image_map example template

#### Why?

If no image set in admin, an error is thrown in twig.

### CODE REVIEW

**Comment by luca-rath** [comment]
@danrot The `media_selection`'s empty value isn't `null` either, it's an empty array

**Comment by alexander-schranz** [comment]
@luca-rath its not about the `media_selection` its about the `single_media_selection` which is null if the image is not set. So this should be the case for the `image_map`, else developers will run always in the same error as you did in the example. 

See the SingleMediaSelection Content Type here: https://github.com/sulu/sulu/blob/aaa1273abefcd3016243766b0199729097dd074b/src/Sulu/Bundle/MediaBundle/Content/Types/SingleMediaSelection.php#L59-L61

File: templates/pages/example.html.twig [hunk 1]
```diff
@@ -85,7 +85,5 @@
     <p>{{ content.contact_account_selection|map(contact => contact.name ?? contact.fullName)|join(', ') }}</p>
     <p>{{ content.text_area }}</p>
 
-    {% if content.image_map %}
```
Comment (alexander-schranz) [inline templates/pages/example.html.twig:88]: This is correct and should work.

**Review by alexander-schranz — CHANGES_REQUESTED** [review]

**Comment by danrot** [comment]
A test is missing.

File: src/Sulu/Bundle/MediaBundle/Content/Types/ImageMapContentType.php [hunk 2]
```diff
@@ -351,12 +351,18 @@ function(ContentTypeInterface $contentType, $property) {
...
     {
-        return $this->prepareData(
+        $data = $this->prepareData(
             $property,
             function(ContentTypeInterface $contentType, $property) {
                 return $contentType->getContentData($property);
             }
         );
+
+        if (!($data['image'] ?? null)) {
```
Comment (danrot) [inline src/Sulu/Bundle/MediaBundle/Content/Types/ImageMapContentType.php:361]: Doesn't `!isset($data['image'])` work as well? This line looks a bit confusing and unusual to me.

**Comment by danrot** [comment]
The CircleCI build is currently also failing on master and this PR is not changing an JS at all... So I am going to merge that as soon as the other tests are passing.

**Review by danrot — APPROVED** [review]

**Pull request merged**